user/api/api.go

```diff
@@ -116,8 +116,7 @@ func NewUsersAPI(userManager *usermanager.UserManager, clientManager *clientmana
 
 func (u *UsersAPI) GetUser(creds Creds, id string) (schema.User, error) {
 	log.Infof("userAPI: GetUser")
-
-	if !u.Authorize(creds) {
+	if creds.User.ID != "" && !u.Authorize(creds) {
```

Comment:
`creds.User.ID != ""`? `u.Authorize` should just recognize if this flag is enabled and perform the correct authentication.